crates/starknet-devnet-types/src/rpc/transactions.rs

```diff
@@ -173,12 +173,12 @@ pub struct TransactionWithReceipt {
     pub transaction: Transaction,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct TransactionStatus {
     pub finality_status: TransactionFinalityStatus,
     pub failure_reason: Option<String>,
-    pub execution_status: TransactionExecutionStatus,
+    pub execution_status: Option<TransactionExecutionStatus>,
```

Comment:
I changed it because the RPC spec for `TXN_STATUS_RESULT` says it is not required. but perhaps this struct is used for more purposes. But we could live with it not being an `Option` since in no place is this property set to `None`.

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/646/commits/de7709335088edf3643318e4c56e155f04c84bb0